### DISCUSSION TITLE
Cannot pair devices SONOFF SNZB-02

### DISCUSSION THREAD

**Discussion by auanasgheps** [Koenkk/zigbee2mqtt#10992] [General]
Hello!
I'm a new user of zigbee2mqtt and I'm facing some issues.
Here's my setup:
- Raspberry Pi4 4GB
- HASSIO / Home Assistant 2021.12
   - Installed on an USB SSD w/ a 1 meter cable to avoid interference
   - Pi is running on good power supply
- zigbee2mqtt + mosquitto (add-ons)
- [RaspBee II](https://phoscon.de/en/raspbee2) running latest firmware (0x26690700)
- Everything is updated/running the latest version

The setup is correct, and I am in fact connected to the RaspBee.
![image](https://user-images.githubusercontent.com/20586878/150804552-7411471e-dc96-4bef-87c5-5dcdfff6ce60.png)

I have purchased two SONOFF SNZB-02, these are my first Zigbee devices.
I was able to pair them with deConz, and with Zigbee2mqtt at least at the beginning.
Using channel 11.

After a day the connection to sensor was lost, and since then I'm unable to pair these devices, they are not discovered at all!
Here's the procedure I use
- Press the sensor button on each sensor until the red light blinks three times
- Click `Permit join (All)` in zigbee2mqtt
- Nothing happens

Does not matter if devices are 10 cm or 10 meters away, they are not seen anymore by the GUI.

To be honest the only thing left to do is to replace the battery, but these devices are new and the led blinks after I press the button.
Am I missing something?

Here are debug logs from the webgui. Nothing:
```
Info Succesfully changed options
Info MQTT publish: topic 'zigbee2mqtt/bridge/response/options', payload '{"data":{"restart_required":false},"status":"ok","transaction":"byz00-1"}'
Debug Received MQTT message on 'zigbee2mqtt/bridge/request/permit_join' with data '{"device":null,"time":254,"transaction":"byz00-2","value":true}'
Info Zigbee: allowing new devices to join.
Info MQTT publish: topic 'zigbee2mqtt/bridge/response/permit_join', payload '{"data":{"time":254,"value":true},"status":"ok","transaction":"byz00-2"}'
Debug Saving state to file /config/zigbee2mqtt/state.json
```

**Comment by miljbee** [comment]
Hello,
I have 10+ of these for several months. This Week, I got one showing the issue you are reporting.
The sensor suddenly reported incorrect temperature and battery, then nothing.
I haven't been able to re pair it, even after replacing the battery.
As I have some spare in stock, I simply picked up a new one, and wrote DEAD on the old one ....

Not sure it helps ... sorry

**Reply by auanasgheps** [reply]
My "original" issue was resolved by going back to DeConz and reset [this](https://github.com/dresden-elektronik/deconz-rest-plugin/issues/3252#issuecomment-691663878) value. 

Then I got back to Z2M and I was able to pair my devices again.

However, one of the two sensors showed up and then died.

I guess these Sonoff are real crap, I will avoid Sonoff zigbee products in the future.

**Reply by amayii0** [reply]
Found this while searching why a SNZB-02 would just stop reporting anything
I bought many batteries, the device flashes red on reset but most won't pair again anymore
So yes, they look crap and die after a year or so :(
